app/views/companies/index.html.erb

```diff
@@ -1,15 +1,14 @@
-<% if params[:query] %>
-  <nav class="breadcrumb">
+<nav class="breadcrumb">
   <%= link_to 'Home', root_path, class: "breadcrumb-item small text-decoration-none" %>
-  <%= link_to 'Stock', companies_path, class: "breadcrumb-item small text-decoration-none" %>
-  <span class="breadcrumb-item active small">Search result</span>
-  </nav>
-<% else %>
-  <nav class="breadcrumb">
-    <%= link_to 'Home', root_path, class: "breadcrumb-item small text-decoration-none" %>
+  <% if params[:query] %>
+    <%= link_to 'Stock', companies_path, class: "breadcrumb-item small text-decoration-none" %>
+    <span class="breadcrumb-item active small">Search result</span>
+  <% elsif params[:stock] %>
```

Comment:
What should you do if I send both parameters (stock and query)?

Reply:
I didn't want to give user possibility of this (this base is too small), but I know user can do it anyway, so I chose to show search result as breadcrumb. I see that controller needs to change order :).

Reply:
But I think I can change it, so app can handle it correctly when it happens, for a future, it is not so much to do